### PR TITLE
Page comment settings

### PR DESCRIPTION
**Pull request by sebastiangreger**
I implemented an additional feature that allows to close comments and/or webmentions for a single page. This is to keep the site admin in control over conversations on their website. It is also possible to hide comments on a single page from being shown on the site.

If comments and/or webmentions are disabled for the applicable template using the global config setting, the toggle appears inactive.

screenshot

This was actually surprisingly straightforward to implement, reusing the UI Kit and some parts of the plugin (I slightly changed the look of the toggle elements since they work differently from the fields they normally appear in: here, no "Save" is needed, as the settings are saved instantly via the API).

### CODE REVIEW

File: src/components/Section.vue [hunk 1]
```diff
@@ -141,6 +156,13 @@ export default {
       this.$store.dispatch("notification/success", ":)");
     },
 
+    async changePageSetting(key, value) {
+      console.log(key + '=' + value);
```
Comment (fabianmichael) [inline src/components/Section.vue:160]: Please remove console.log before merge

File: src/components/PageSettingsToggle.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+</script>
+
+<style lang="scss">
+.k-commentions-pagesettings .k-toggle-input .k-toggle-input-label {
+  font-size: .875rem;
+}
+.k-commentions-pagesettings .k-toggle-input[data-disabled="true"] .k-toggle-input-label {
+  color: #ccc;
+}
+.k-commentions-pagesettings .k-toggle-input-native:not([disabled]) {
```
Comment (fabianmichael) [inline src/components/PageSettingsToggle.vue:57]: I’m a bit undecided, whether I like the colored toggle buttons. They don’t look very Kirby-like, but are in fact very helpful! :-) Maybe we could move them to a settings-sidebar or the header in a futute release. Let’s keep them this way for now. I also thought about mass-editing already, but this is a whole different story, that we should not tackle by now.

File: src/components/Section.vue (hunk 1 above)
Comment (sebastiangreger) [reply src/components/Section.vue]: Good catch! :heavy_check_mark:

File: src/components/PageSettingsToggle.vue (hunk 2 above)
Comment (sebastiangreger) [reply src/components/PageSettingsToggle.vue]: I hear you - this was an exploration, and I felt the same. I removed the colours for now - can be revisited at a later stage.

**Comment by fabianmichael** [comment]
@sebastiangreger I think this can be merged then.

**Pull request merged**